platforms/windows/src/util.rs

```diff
@@ -116,7 +116,7 @@ pub(crate) fn safe_array_from_i32_slice(slice: &[i32]) -> *mut SAFEARRAY {
     safe_array_from_slice(VT_I4, slice)
 }
 
-pub(crate) enum Event {
+pub(crate) enum QueuedEvent {
```

Comment:
I couldn't tell at a glance whether the new `windows::core::Event` type might be able to replace this type, so went with a rename for now.

Reply:
The rename is definitely a good solution. Thanks.